README.md

```diff
@@ -58,6 +58,13 @@ As this is early stage work, there's lots improvements that can be done in the f
 
 Here's a quick run through on what you need to know to use `prr` effectively.
 
+### I just want to use it
```

Comment:
Maybe let's restructure the readme a little bit to make it clear that you can just use "pip install" ---OR--- do all the other stuff.